.gitignore

```diff
@@ -1,4 +1,5 @@
-out/
-node_modules/
 .DS_Store
+
+/out/
+/node_modules/
```

Comment:
Can you explain why there was a need to move these lines?

Reply:
Force of habbit - `.DS_Store` first. Have reverted the order, kept the leading slashes to home/hone the ignores to the root.